dear_petition/petition/etl/load.py

```diff
@@ -48,6 +48,15 @@ def create_batch_petitions(batch):
         )
         link_offense_records_and_attachments(petition)
     # TODO: Not guilty
+    not_guilty_records = batch.not_guilty_offense_records()
+    petition_types = identify_distinct_petitions(not_guilty_records)
+    for petition_type in petition_types:
+        petition = batch.petitions.create(
+            form_type=NOT_GUILTY,
+            jurisdiction=petition_type["jurisdiction"],
+            county=petition_type["county"],
+        )
+        link_offense_records_and_attachments(petition)
```

Comment:
Can you make a separate function that performs this logic? Something like

```
def create_petitions_from_records(records):
    petition_types = identify_distinct_petitions(records)
    for petition_type in petition_types:
        petition = batch.petitions.create(
            form_type=NOT_GUILTY,
            jurisdiction=petition_type["jurisdiction"],
            county=petition_type["county"],
        )
        link_offense_records_and_attachments(petition)

...
create_petitions_from_records(batch.dismissed_offense_records())
create_petitions_from_records(batch.not_guilty_offense_records())
```

Reply:
Yep, good idea.